src/Vehicle.java

```diff
@@ -0,0 +1,29 @@
+public abstract class Vehicle {
+
+    private String modelName;
+    private int wheelsCount;
+
+    public Vehicle(String modelName, int wheelsCount) {
+        this.modelName = modelName;
+        this.wheelsCount = wheelsCount;
+    }
+
+    public String getModelName() {
+        return modelName;
+    }
+
+    public int getWheelsCount() {
+        return wheelsCount;
+    }
+
+    public void setModelName(String modelName) {
+        this.modelName = modelName;
+    }
+
+    public void setWheelsCount(int wheelsCount) {
+        this.wheelsCount = wheelsCount;
+    }
+
+    public void check() {
+    }
```

Comment:
почему метод пустой? его надо абстрактным сделать, а не пустым
```suggestion
    public abstract void check();
```